.github/workflows/pr.yml

```diff
@@ -0,0 +1,160 @@
+name: PR Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+env:
+  REALM_CI: true
+
+jobs:
+  baas-linux:
+    runs-on: ubuntu-latest
+    name: BaaS Linux
+    outputs:
+      clusterName: ${{ steps.deploy-mdb-apps.outputs.clusterName }}
+    timeout-minutes: 20
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: false
+      - uses: realm/ci-actions/mdb-realm/deployApps@fac1d6958f03d71de743305ce3ab27594efbe7b7
+        id: deploy-mdb-apps
+        with:
+          projectId: ${{ secrets.ATLAS_PROJECT_ID }}
+          apiKey: ${{ secrets.ATLAS_PUBLIC_API_KEY }}
+          privateApiKey: ${{ secrets.ATLAS_PRIVATE_API_KEY }}
+          differentiator: dart-linux
```

Comment:
The name is fairly irrelevant - it gets hashed in combination with some other variables. As long as each job in a workflow has a unique name, it doesn't really matter what that name is.

Reply:
could we rename to `realm-dart` to avoid confusion

Reply:
No, because we're going to have tests for different platforms - e.g. linux, macOS, windows. They need to each deploy their own BaaS server and run tests against them. So we'll have `dart-linux`, `dart-macos`, etc. This also enables us to use the same naming scheme if we want to have flutter tests on desktop platforms - e.g. `flutter-linux`, etc. I don't think we need to run all those tests on each PR build as that will slow things down, but might be a good idea to run them on every merge to master.

Reply:
I see, thanks.